huaweicloud/services/aom/resource_huaweicloud_aom_alarm_silence_rule.go

```diff
@@ -0,0 +1,454 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product AOM
+// ---------------------------------------------------------------
+
+package aom
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceAlarmSilenceRule() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAlarmSilenceRuleCreate,
+		UpdateContext: resourceAlarmSilenceRuleUpdate,
+		ReadContext:   resourceAlarmSilenceRuleRead,
+		DeleteContext: resourceAlarmSilenceRuleDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the rule name.`,
+			},
+			"time_zone": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the time zone.`,
+			},
+			"silence_time": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Elem:     AlarmSilenceRuleSilenceTimeSchema(),
+				Required: true,
+			},
+			"silence_conditions": {
+				Type:        schema.TypeList,
+				Elem:        AlarmSilenceRuleSilenceConditionsSchema(),
+				Required:    true,
+				Description: `Specifies the silence conditions of the rule.`,
+			},
+			"description": {
```

Comment:
the description in console can update to empty. Please check if the field need to mark as computed

Reply:
as discussed, keep it as it is